react-app/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_BASE_URL=http://localhost:3000
```

Comment:
Dont push the .env to github. It will normally have very sensitive information

Reply:
Ok, understood!